src/ffi/types.rs

```diff
@@ -54,7 +54,10 @@ pub use crate::logger::{LogLevel, LogRecord, LogWriter};
 pub use crate::payment::store::{
 	ConfirmationStatus, LSPFeeLimits, PaymentDirection, PaymentKind, PaymentStatus,
 };
-pub use crate::payment::QrPaymentResult;
+pub use crate::payment::UnifiedPaymentResult;
+
+pub use lightning::onion_message::dns_resolution::HumanReadableName as LdkHumanReadableName;
```

Comment:
I don't think we want to expose the `LdkHumanReadableName` variant, this is just an internal type alias, no?